kubernetes_env/kube_env.py

```diff
@@ -14,10 +14,13 @@
 
 FILE_DIR = Path(__file__).parent.resolve()
 ROOT_DIR = FILE_DIR.parent
-ISTIO_DIR = FILE_DIR.joinpath("istio-1.9.1")
+ISTIO_DIR = FILE_DIR.joinpath("istio-1.9.3")
 ISTIO_BIN = ISTIO_DIR.joinpath("bin/istioctl")
 YAML_DIR = FILE_DIR.joinpath("yaml_crds")
 TOOLS_DIR = FILE_DIR.joinpath("tools")
+ONLINE_BOUTIQUE_DIR = FILE_DIR.joinpath("microservices-demo")
```

Comment:
It may make sense to turn this into a config matrix. So for each application you have a set of configuration options that you just need to get from the map. Then you do not have to add all these constants and can just extend the map, if needed.